inspec.lock

```diff
@@ -0,0 +1,3 @@
+---
```

Comment:
Remove the `inspec.lock` file as this is automatically generated on every new run.

Reply:
scrubbed